homeassistant/components/stookwijzer/diagnostics.py

```diff
@@ -0,0 +1,19 @@
+"""Diagnostics support for Stookwijzer."""
+from __future__ import annotations
+
+from typing import Any
+
+from stookwijzer import Stookwijzer
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    client: Stookwijzer = hass.data[DOMAIN][entry.entry_id]
+    return {"state": client.state}
```

Comment:
The state by itself is already in the entity, what does the diagnostic add in this case?
And why add only the state and not the rest of the data response?